### PR TITLE
KAIZ 128 Room Creation with 100ms

### PR DESCRIPTION
**Pull request by jasjot1**
* Meeting post will also do a post request on 100ms api to create a room id
* Take that room ID and then do another post request to get room code
* The room code and room id is then stored with the other meeting data
* Room code is passed into MeetingVideo to join the meeting

### CODE REVIEW

File: server/api/meeting/index.post.ts [hunk 1]
```diff
@@ -19,32 +43,51 @@ export default defineEventHandler(async (event) => {
...
 
-		// Check if meeting end time is greater than start time
-		if (endTime < startTime) {
-			throw createError({
-				statusCode: 400,
-				statusMessage: 'The meeting end time must be after the start time.',
-			})
-		}
+		// Create room in 100ms
+		const roomResponse = await fetch('https://api.100ms.live/v2/rooms', {
```
Comment (sparshs916) [inline server/api/meeting/index.post.ts:52]: Wondering if we should wrap this in a try catch in case there is an error?

**Review by sparshs916 — APPROVED** [review]

File: server/api/meeting/index.post.ts (hunk 1 above)
Comment (ddenaga) [reply server/api/meeting/index.post.ts:30]: It's already wrapped in a try catch, but I'm not sure if an HMS error will be handled correctly.

File: nuxt.config.ts [hunk 2]
```diff
@@ -37,5 +37,7 @@ export default defineNuxtConfig({
 		FAUNA_KEY: process.env.FAUNA_KEY,
 		AUTH_SECRET: process.env.AUTH_SECRET,
 		AUTH_ORIGIN: process.env.AUTH_ORIGIN,
+		APP_ACCESS_KEY: process.env.APP_ACCESS_KEY,
```
Comment (ddenaga) [inline nuxt.config.ts:40]: Can we rename them so it's clear they're HMS environment variables?

File: server/api/meeting/index.post.ts (hunk 1 above)
Comment (jasjot1) [reply server/api/meeting/index.post.ts:30]: Should handle the hms errors now

**Review by ddenaga — APPROVED** [review]
LGTM

**Pull request merged**